diff -urN clutter-gst-0.10.0/configure.ac clutter-gst-0.10.0.new/configure.ac
--- clutter-gst-0.10.0/configure.ac	2009-07-29 12:12:32.000000000 +0200
+++ clutter-gst-0.10.0.new/configure.ac	2009-11-08 19:09:21.728006779 +0100
@@ -107,11 +107,9 @@
 
 GST_MAJORMINOR=0.10
 
-pkg_modules="gstreamer-$GST_MAJORMINOR gstreamer-plugins-base-$GST_MAJORMINOR gstreamer-base-$GST_MAJORMINOR"
+pkg_modules="gstreamer-$GST_MAJORMINOR gstreamer-base-$GST_MAJORMINOR"
 PKG_CHECK_MODULES(GST, [$pkg_modules])
 
-GST_LIBS="$GST_LIBS -lgstinterfaces-$GST_MAJORMINOR -lgstvideo-$GST_MAJORMINOR -lgstaudio-$GST_MAJORMINOR"
-
 dnl ========================================================================
 
 m4_define([maintainer_flags_default], [yes])
